akd/benches/azks.rs

```diff
@@ -55,6 +55,9 @@ fn batch_insertion(c: &mut Criterion) {
                 let db = StorageManager::new_no_cache(&database);
                 let mut azks = runtime.block_on(Azks::new(&db)).unwrap();
 
+                // create transaction object
```

Comment:
so one consideration here, is we are never "committing" the transaction, i.e. paying the price to write it to the underlying DB. This means all the data will effectively be dropped, but imho this is fine for this benchmark since we are not wanting to compare the db contention time for reads/writes, but rather the amount of time spent hashing + interacting in parallel with the transaction struct.